final_prep/annotations.rb

```diff
@@ -4,40 +4,74 @@
 
 # Build a Bear
 
+# Declare a method named build_a_bear that takes the arguments name, age, fur, clothes, and special_power
 def build_a_bear(name, age, fur, clothes, special_power)
+  # assign the variable greeting to a string with the name argument interpolated inside
   greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+  # assign the variable demographics to an array consisting of the arguments name and age
   demographics = [name, age]
+  # assign the variable power_saying to a string with the special_power argument interpolated inside
   power_saying = "Did you know that I can #{special_power}?"
+  # assign the variable built_bear to a hash
   built_bear = {
+    # assign the key basic_info the value of the variable demographics
     'basic_info' => demographics,
+    # assign the key clothes the value of the argument clothes
```

Comment:
Nice- way to specify whether the values come from an argument or variable declared above.